tests/system/Config/fixtures/.env

```diff
@@ -6,4 +6,5 @@ NULL=
 
 SimpleConfig.onedeep=baz
 SimpleConfig.default.name=ci4
-simple.name=simpleton
\ No newline at end of file
+simple.name=simpleton
+impleconfig.shortprefix=wht
```

Comment:
If you add the S I thing the test will fail. This is actually interesting part if you look at how it define the short prefix you will know why. 
` $shortPrefix = strtolower(substr($prefix, strrpos($prefix, '\\') + 1));`
So basically, if you have "SimpleConfig" the short prefix will remove "S" and then lower case rest of letters. "SimpleConfig" -> "impleconfig"